BridgeAppSDK/Localization.swift

```diff
@@ -154,92 +156,6 @@ public class Localization: NSObject {
     
     // MARK: List of the strings used in this module.
     public class var listOfAllLocalizedStrings: [String : String] {
-        return listOfMyLocalizedStrings
+        return [:]
```

Comment:
Should we maybe just do away with this method and go with the .strings file in all the modules that subclass this (e.g. mPowerSDK, various apps)? If so, would be better to do so now before anyone else sees/follows this pattern...